.tool-versions

```diff
@@ -2,4 +2,5 @@ erlang 22.2.1
 erlang 22.2.4
 elixir 1.9.4
 elixir 1.9.4-otp-22
+helm 3.1.1
```

Comment:
I believe @jmorris-pillar found a breaking change from `3.0` to `3.1`.

Reply:
any idea what it was? i didn't see any issues.

Reply:
You'll have to ask him. I can't remember for the life of me.

Reply:
i'll leave it, as this seems to be resolved after discussing (issues with the `required` macro that i didn't encounter); either version should be fine.